cico_setup.sh

```diff
@@ -37,13 +43,13 @@ push_image() {
     image_name=$(make get-image-name)
     image_repository=$(make get-image-repository)
     short_commit=$(git rev-parse --short=7 HEAD)
-    push_registry="push.registry.devshift.net"
+    #push_registry="push.registry.devshift.net"
     # Only for testing
-    push_registry="hub.docker.com"
+    push_registry="docker.io"
```

Comment:
Please leave these changes to #38

Reply:
Hey, @anmolbabu , I get a 404 from docker when I use hub.docker.com/devanggaur7/supervisor as the image tag. It only works for me with docker.io . I can't test it locally otherwise.

Reply:
Ok I didn't hit that issue and this worked for me.

Also, I only meant that after the script is verified and the PR is ready for merge, please retain back the old state of these lines as it would later be changed as in https://github.com/fabric8-hdd/supervisor/pull/38 once that's merged..